packages/@aws-cdk/aws-lambda/lib/lambda-role.ts

```diff
@@ -0,0 +1,127 @@
+import { IManagedPolicy, ManagedPolicy, Role, RoleProps, ServicePrincipal } from '@aws-cdk/aws-iam';
+import { Construct} from '@aws-cdk/core';
+
+/**
+ * Properties for defining a Lambda Role
+ */
+export interface LambdaRoleProps extends Omit<RoleProps, 'assumedBy'> {
+  // We omit the assumedBy property because we know what it is for a Lambda Role
+
+  /**
+   * Grants permission to manage elastic network interfaces to
```

Comment:
Perhaps be transparent and say

```suggestion
   * Adds the AWS managed policy "AWSLambdaBasicExecutionRole" to this role
```

Reply:
Done